packaging/debian-sid/tests/integrationtests

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+set -ex
+
+# required for the debian adt host
+mkdir -p /etc/systemd/system/snapd.service.d/
+if [ "${http_proxy:-}" != "" ]; then
+    cat <<EOF | tee /etc/systemd/system/snapd.service.d/proxy.conf
+[Service]
+Environment=http_proxy=$http_proxy
+Environment=https_proxy=$http_proxy
+EOF
+
+    # ensure environment is updated
+    echo "http_proxy=$http_proxy" >> /etc/environment
+    echo "https_proxy=$http_proxy" >> /etc/environment
+fi
+systemctl daemon-reload
+
+# ensure we are not get killed too easily
+printf '%s\n' "-950" > /proc/$$/oom_score_adj
+
+# see what mem we have (for debugging)
+cat /proc/meminfo
+
+# ensure we can do a connect to localhost
+echo ubuntu:ubuntu|chpasswd
```

Comment:
I didn't notice this before. Will it harm us eventually once Debian adt runner is updated and test suddenly start to execute?

Reply:
We use the following in debian/tests/control: `Restrictions: ..., breaks-testbed, isolation-machine` - ie this makes it clear that the tests will break the machine and a snapshot (or similar) needs to be restored once the tests are finished.